### DISCUSSION TITLE
Display window when modifier key is held

### DISCUSSION THREAD

**Discussion by GetPsyched** [elkowar/eww#957] [Q&A] [answered]
^ How can I let the eww window be hidden by default and only shown when I hold certain key(s), say, `Mod4`.

Couldn't find other discussions on this; sorry if it's a duplicate.

**Comment by w-lfchen** [comment, marked as answer]
This can't be solved with eww alone, I'd suggest looking into some hotkey software or what your window manager supports.
if you have something that can toggle on a button being pressed and released, you could use `eww open --toggle <your-window-name>` on both of those.

**Reply by GetPsyched** [reply]
Thanks. I figured that might be my only way out.
